packages/server/src/supertokens.test.ts

```diff
@@ -141,7 +140,6 @@ describe("supertokens", () => {
 
       const publicData = JSON.parse(publicDataStr)
       expect(publicData.userId).toBe(1)
-      expect(publicData.roles[0]).toBe("admin")
```

Comment:
Leave this test in here. We still want this to work, even though it won't be required.